api/proto/security/authn.proto

```diff
@@ -289,3 +292,20 @@ message Iap {
   // The URL containing the Cloud IAP public keys in JWK format.
   string iapVerifyKeyUrl = 5;
 }
+
+// Configuration for basic username/password authentication
+message Basic {
```

Comment:
> Users *will* need to
move any basic auth config from the service settings file to the
hal config.

Should we add this sentence to the list of breaking changes in the README?

Reply:
Yes! Definitely, and I also need to move the few other things in our Google doc to the README.   I probably should have done it in this PR, but I'll do it in the next PR that adds unit tests.